.github/workflows/docsbuild-release.yaml

```
@@ -0,0 +1,85 @@
+name: Create an Ansible Release Porting Guide PR
+on:
+  workflow_dispatch:
+    inputs:
+      ansible-version:
+        description: >-
+          Release Version. Example: 11.1.0
+        required: true
+env:
+  CI_COMMIT_MESSAGE: >-
+    Add Ansible community "${{ inputs.ansible-version }}" porting guide
+
+jobs:
+  build:
+    name: >-
+      Build Porting-Guide (${{ inputs.ansible-version }})
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - name: Retrieve the major version component
+        id: compute-version
+        run: |
+          echo -n 'major-version=' >> "${GITHUB_OUTPUT}"
+          echo '${{ inputs.ansible_version }}' | sed 's#\([0-9]\+\)\..*#\1#' >> "${GITHUB_OUTPUT}"
+
+      - name: Check out ansible-documentation
+        uses: actions/checkout@v3
+        with:
+          path: ansible-documentation
+          ref: devel
+
+      - name: Checking out to a new branch & setting the user details
+        working-directory: ansible-documentation
+        run: |
+          git checkout -b "publish-${{ inputs.ansible-version }}"
+          git config --global user.name "Github Actions"
+          git config --global user.email "41898282+github-actions[bot]@users.noreply.github.com"
+
+# Get the Porting guide from the ansible-build-data git repo
+
+      - name: Check out ansible-build-data
+        uses: actions/checkout@v3
+        with:
+          repository: ansible-community/ansible-build-data
+          ref: main
+          path: ansible-build-data
+
+      - name: Copy the porting guide document to ansible-documentation
+        run: >-
+          cp -v
+          'ansible-build-data/${{ steps.compute-version.outputs.major-version }}/porting_guide_${{ steps.compute-version.outputs.major-version }}.rst'
+          './ansible-documentation/docs/docsite/rst/porting_guides/'
+
+# the actual steps for commit
+
+      - name: >-
+          Create a pull request branch with the changes
+          in the ansible-documentation GitHub repository
+        working-directory: ansible-documentation
+        run: |
+         git add ./docs/docsite/rst/porting_guides/"porting_guide_${{ steps.compute-version.outputs.major-version }}.rst"
+         git commit -m "${CI_COMMIT_MESSAGE}"
+         git push origin "publish-${{ inputs.ansible-version }}"
+        env:
+         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

these are underindented
```suggestion
          git add ./docs/docsite/rst/porting_guides/"porting_guide_${{ steps.compute-version.outputs.major-version }}.rst"
          git commit -m "${CI_COMMIT_MESSAGE}"
          git push origin "publish-${{ inputs.ansible-version }}"
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```